chordspy/connection.py

```
@@ -0,0 +1,681 @@
+        def notification_handler(sender, data):
+            if len(data) == self.ble_connection.NEW_PACKET_LEN:
+                if not self.lsl_connection:
+                    self.setup_lsl(num_channels=3, sampling_rate=500)
+                
+                original_notification_handler(sender, data)
+                
+                for i in range(0, self.ble_connection.NEW_PACKET_LEN, self.ble_connection.SINGLE_SAMPLE_LEN):
+                    sample_data = data[i:i+self.ble_connection.SINGLE_SAMPLE_LEN]
+                    if len(sample_data) == self.ble_connection.SINGLE_SAMPLE_LEN:
+                        channels = [int.from_bytes(sample_data[i:i+2], byteorder='big', signed=True) 
+                            for i in range(1, len(sample_data), 2)]
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)